sdk/src/main/java/io/radar/sdk/RadarApiHelper.kt

```diff
@@ -30,8 +30,13 @@ internal open class RadarApiHelper(
                          headers: Map<String, String>?,
                          params: JSONObject?,
                          sleep: Boolean,
-                         callback: RadarApiCallback? = null) {
-        logger?.d("📍 Radar API request | method = ${method}; url = ${url}; headers = ${headers}; params = $params")
+                         callback: RadarApiCallback? = null,
+                         stream: Boolean = false) {
+        if (stream) {
+            logger?.d("📍 Radar API request | method = $method; url = $url; headers = $headers; params = <stream>")
```

Comment:
Reflecting on this more, I see now how important this change is.

Otherwise, if we add a log entry for the for `/v1/log` api calls (and it contains the full `params` payload), we'll always run out of memory eventually.  Because the log payloads will never get completely removed from the buffer, they'll just get coalesced into this `params` attribute here, which will grow during each round of flushing to the server. Eventually we'll just run out of memory.

@phil-brown check my reasoning here to make sure it's right.

cc @corypisano @jrtibbetts in case we need to bring this to iOS.  Could this be what is causing the crashes on iOS? 🤔 

Also, instead of reusing the `stream` param here to change how we log the params, I think it's worth being explicit and adding another param, eg, `logPayload: Boolean = false`.  Then, add a comment at the call site in `RadarApiClient.log()` as to why, since it's a critical piece of code.

Reply:
@tjulien that's right. I have discussed this fix with @corypisano and @jrtibbetts on slack as well. I think adding an additional param makes a lot of sense!

Reply:
@tjulien updated. Added some clarification comments in the code.